api/src/config/logger.py

```diff
@@ -2,18 +2,19 @@
 Create the logger to be used in the service
 """
 
-import os
+from os import getenv
 
 from arkhn_monitoring import create_logger
 
 
-FLUENTD_HOST = os.getenv("FLUENTD_HOST", "fluentd")
-FLUENTD_PORT = os.getenv("FLUENTD_PORT", 24224)
+FLUENTD_HOST = getenv("FLUENTD_HOST", "fluentd")
+FLUENTD_PORT = getenv("FLUENTD_PORT", 24224)
+LOGGING_LEVEL = getenv("LOGGING_LEVEL", "INFO")
```

Comment:
Cool, that way we can choose how much logging we want to store?

Reply:
Exactly, we just have to change an environment variable to do so